tests/Paramore.Brighter.RMQ.Async.Tests/MessagingGateway/Proactor/When_a_message_consumer_a_cloud_events.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Threading.Tasks;
+using Paramore.Brighter.MessagingGateway.RMQ.Async;
+using Xunit;
+
+namespace Paramore.Brighter.RMQ.Async.Tests.MessagingGateway.Proactor;
+
+[Trait("Category", "RMQ")]
+public class RMQBufferedConsumerCloudEventsTestsAsync : IAsyncDisposable 
+{
+    private readonly IAmAMessageProducerAsync _messageProducer;
+    private readonly IAmAMessageConsumerAsync _messageConsumer;
+    private readonly ChannelName _channelName = new(Guid.NewGuid().ToString());
+    private readonly RoutingKey _routingKey = new(Guid.NewGuid().ToString());
+    private const int BatchSize = 3;
+
+    public RMQBufferedConsumerCloudEventsTestsAsync()
+    {
+        var rmqConnection = new RmqMessagingGatewayConnection
+        {
+            AmpqUri = new AmqpUriSpecification(new Uri("amqp://guest:guest@localhost:5672/%2f")),
+            Exchange = new Exchange("paramore.brighter.exchange")
+        };
+
+        _messageProducer = new RmqMessageProducer(rmqConnection);
+        _messageConsumer = new RmqMessageConsumer(connection: rmqConnection, queueName: _channelName,
+            routingKey: _routingKey, isDurable: false, highAvailability: false, batchSize: BatchSize);
+
+        //create the queue, so that we can receive messages posted to it
+        new QueueFactory(rmqConnection, _channelName, new RoutingKeys(_routingKey)).CreateAsync().GetAwaiter()
+            .GetResult();
+    }
+
+    [Fact]
+    public async Task When_a_message_consumer_a_cloud_events_async()
```

Comment:
Minor, but the name of this test does not parse. Do you means uses_cloud_events_async ?